circuitClient.js

```diff
@@ -139,16 +143,14 @@ class CircuitClient {
       return Promise.resolve();
     }
     const displayName = this.client.loggedOnUser.displayName;
-    return this.client.logout()
-      .then(_ => {
-        console.log(`Logged out of Circuit: ${displayName}`)
-      });
+    return this.client.logout().then(_ => {
+      console.log(`Logged out of Circuit: ${displayName}`);
+    });
   }
 
   /////////////////////////////////////
   /// Private functions
   /////////////////////////////////////
-
 }
 
-module.exports = CircuitClient;
\ No newline at end of file
+module.exports = CircuitClient;
```

Comment:
Our styleguide rules from eslint also requires a newline at the end of the file

Reply:
Done